### DISCUSSION TITLE
Any ways to change the remote .bin file?

### DISCUSSION THREAD

**Discussion by glaypeng78** [GerryDazoo/Slinger#332] [Announcements]
I have a slingbox 500 and work well with the Slinger. Really appreciate all your great works. However I got trouble while I decide to change my cable box provider. I get used to configure the remote function with the .bin file, for me it will be like C2010_PL.bin. Is it possible to configure new .bin file while the slingbox official server is already shut down?

**Comment by GerryDazoo** [comment]
No it's not possible yet. We're working on it but it might take awhile. If you're desperate there are other third party remote IR solutions that you can buy.
Please make sure your 500 is properly firewalled. I personally believe you're taking a big risk using the Sling supplied software after N9 to make any configuration changes,

**Reply by glaypeng78** [reply]
Thanks for your remind.  Actually I just test my 500 with your Slinger and soon unplug it around maybe N7. After that I just leave it there and wait for my new TV cable provider to set the TV box at my house (in order to watch World Cup). Recently I am ready to set the Slinger again and while I visit your project to see if there's any update I saw many people encounter the brick problem. I firewalled at the first moment and I am using a third party remote IR call "Broadlink RM4mini" now. I am not a professional in this field so if I was wrong please be understood. Maybe your team can try to flash the bios to solve the bricked Slingbox. I get used to fix the problem for some of my friends' laptop by flashing the bios with solder. Just hope this kind of opinion can help and looking forward to see your team to make Slinger even much better.
